content/articles/using-machine-learning-to-translate-brain-signals-to-text/index.md

```diff
@@ -0,0 +1,136 @@
+
+In this article, we will explore how machine learning is used to decode and translate brain neural signals into handwritten texts. Indeed, it is a first of its kind. Previous BCI research has focused on restoring motor skills such as reading and grasping. Not much focus has been put in researching the use of BCI to solve complex behavioral tasks. This article aims to tackle that. 
+
+The researchers in this study worked with a paralysis patient with a spinal code injury. He was paralyzed from the neck downwards with his hand movements completely non-functional. The goal was to make the patient attempt to imagine writing with a pen on a piece of paper as if his hands weren't paralyzed. 
+
+![Patient with electrode implants imagining handwriting](/engineering-education/using-machine-learning-to-translate-brain-signals-to-text/electrode-implants.PNG)
+
+*[Image Source: HHMI](https://www.hhmi.org/news/brain-computer-interface-turns-mental-handwriting-into-text-on-screen)*
+
+The figure shows two tiny arrays of implanted electrodes relaying information from the brain area of the patient. It controls the hands and arms to an algorithm. The algorithm then translates these signals into letters that appear on a screen. 
+
+Let's try and understand how they do this.
+
+### Neural representation of handwriting
+
+To visualize the neural activities recorded during the attempted writing from the patient's brain, two machine learning techniques were used:
+
+1. Principal Component Analysis (PCA) is one of the methods for dimensionality reduction. It takes your data and extracts the components that have the highest amount of variance. The goal of PCA in this experiment was to extract the top three neural signals (components) associated with handwriting. It also eliminated things such as background noise or things that might not be relevant to our task.
+
+![PCA](/engineering-education/using-machine-learning-to-translate-brain-signals-to-text/pca.PNG)
+
+*[Image Source: Nature](https://www.nature.com/articles/s41586-021-03506-2)*
+
+The above example shows the neural activity in the top 3 principal components for three example letters (d, e, and m).
+
+2. They used a non-linear dimensionality reduction technique called t-distributed stochastic neighbor embedding (t-SNE). The method showed that data can be clustered into different groups representing different letters of the alphabet (a-z). This showed that the data could indeed be separable. Thus, this method helped to distinguish neural activities in the brain associated with different letters i.e., the letter 'a' and the letter 'l'.
+
+![t-SNE](/engineering-education/using-machine-learning-to-translate-brain-signals-to-text/t-sne.PNG)
+
+*[Image Source: Nature](https://www.nature.com/articles/s41586-021-03506-2)*
+
+The figure represents the 2D visualization of the neural activity made using t-SNE. 
```

Comment:
A keen reader will note the two techniques and naturally ask, why are dimensionality reduction techniques used here in the first place?